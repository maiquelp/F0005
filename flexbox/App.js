import React from 'react';
import {
  StyleSheet,
  Text,
  View,
} from 'react-native';

const App = () => {
  return (

        <View style={styles.container}>
          <View style={styles.header}>
            <Text>Back</Text>
            <Text>Title</Text>
            <Text>Profile</Text>
          </View>
        </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#CCC'
  },
  header: {
    height: 60,
    backgroundColor: '#FFF',
    paddingHorizontal: 20,
    paddingTop: 20,
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'space-between'
  }
});

export default App;
