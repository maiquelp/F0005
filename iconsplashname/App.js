import React from 'react';
import {
  StatusBar,
  StyleSheet,
  Text,
  View,
} from 'react-native';

const App = () => {
  return (
    <View style={styles.container}>
    <StatusBar
      barStyle="light-content"
      backgroundColor={styles.container.backgroundColor}
    />
    <Text style={styles.welcome}>Welcome!</Text>
  </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#7159C1"
  },
  welcome: {
    fontSize: 20,
    textAlign: "center",
    margin: 10,
    color: "#FFF",
    fontWeight: "bold"
  }
});

export default App;
