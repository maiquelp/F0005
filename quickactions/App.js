import React from 'react';
import {Platform, DeviceEventEmitter, StyleSheet, Text, View} from "react-native";
import QuickActions from "react-native-quick-actions";

QuickActions.setShortcutItems([
  {
    type: "contacts", // Required
    title: "Listar contatos", // Optional, if empty, `type` will be used instead
    subtitle: "Ver amigos",
    icon: Platform.OS === "ios" ? "Rocket" : "icon", // Pass any of UIApplicationShortcutIconType<name>
    userInfo: {
      url: "app://contacts" // provide custom data, like in-app url you want to open
    }
  }
]);

DeviceEventEmitter.addListener("quickActionShortcut", data => {
  console.log(data);
});

QuickActions.popInitialAction().then(data => {
  console.log(data);
});

const App = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.welcome}>Welcome to React Native!</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#F5FCFF"
  },
  welcome: {
    fontSize: 20,
    textAlign: "center",
    margin: 10
  }
});

export default App;
