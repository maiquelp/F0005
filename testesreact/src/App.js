import {Provider} from 'react-redux';
import store from './store';

import Todos from './Todos';

function App() {
  return (
    <Provider store={store}>
      <div className="App">
          <p>
            <h1>Edit <code>src/App.js</code> and save to reload.</h1>
            <Todos />
          </p>
      </div>
    </Provider>
  );
}

export default App;
