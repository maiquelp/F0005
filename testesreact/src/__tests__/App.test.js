import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Testing App', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <App />
    );
    expect(wrapper).toMatchSnapshot();
  });
});