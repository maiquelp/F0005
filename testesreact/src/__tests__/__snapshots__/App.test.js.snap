// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App should render properly 1`] = `ShallowWrapper {}`;
